src/main/java/com/microsoft/sqlserver/jdbc/KeyVaultCredential.java

```diff
@@ -24,20 +24,33 @@
  */
 class KeyVaultCredential extends KeyVaultCredentials {
 
-    String clientId = null;
-    String clientKey = null;
+    SQLServerKeyVaultAuthenticationCallback authenticationCallback = null;
+    String                                  clientId               = null;
+    String                                  clientKey              = null;
+    String                                  accessToken            = null;
```

Comment:
formatting here?

Reply:
That's how mssql-jdbc formatter works here.